packages/common/src/store/notifications/types.ts

```diff
@@ -104,8 +104,230 @@ export type BaseNotification = {
   isViewed: boolean
   timestamp: string
   timeLabel?: string
+  // group id is a part of the notifications v2 spec
+  groupId?: string
 }
 
+export type DiscoveryBaseNotification<T, P> = {
+  type: T
+  group_id: string
+  seen_at: number | null
+  actions: DiscoveryAction<P>[]
+}
+
+export type DiscoveryAction<T> = {
+  specifier: string
+  type: string
+  timestamp: number
+  data: T
+}
+
+export type DiscoveryAnnouncementNotificationAction = {
```

Comment:
sweet thanks for adding all these types, def curious if these can become the base of the new notification types?